fighthealthinsurance/common_view_logic.py

```
@@ -3004,6 +3006,22 @@ async def generate_appeals(cls, parameters) -> AsyncIterator[str]:
+            # A live human outranks any background generator: STEAL the
+            # denial's generation lease so a journey attempt in flight sees
+            # the epoch move and stops quietly, and one arriving inside the
+            # TTL backs off. One UPDATE; expiry is the release. Never let a
+            # lease hiccup break the interactive flow (external review).
+            try:
+                await generation_lease.aacquire(
+                    denial,
+                    holder=generation_lease.new_holder("interactive"),
+                    steal=True,
+                )
+            except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)